invenio_cli/helpers/commands.py

```diff
@@ -333,6 +333,25 @@ def _lock_python_dependencies(self, pre):
         if not locked:
             subprocess.run(command, check=True)
 
+    def destroy(self):
+        """Destroys a virtualenv (if created with pipenv) and containers."""
```

Comment:
major: Try to *never repeat code*. If something is repeated, it means *it has to be extracted into a function*. My point being, this and `LocalCommands.destroy` is 100% the same, should be written only once.

Reply:
We ended up putting "very similar" code in LocalCommands and ContainerizedCommands to get things moving. We can refactor this later when the higher-level commonality is even clearer.